frontend/public/index.html

```diff
@@ -2,8 +2,23 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Chillzone is a student-friendly platform designed to help IIT Kharagpur students locate empty classrooms during class hours. By providing real-time availability of classrooms across different complexes, Chillzone empowers students to find their perfect study or relaxation spot. Whether you need a quiet place to focus or a space to unwind between classes, Chillzone has you covered."
```

Comment:
I agree with Shikhar. Please also remove "real-time" because chillzone is static and doesn't change in real time.

Reply:
changed the description. please review